internal/v1/api.yaml

```diff
@@ -947,6 +947,8 @@ components:
           type: boolean
         check_repo_gpg:
           type: boolean
+        enabled:
+          type: boolean
```

Comment:
Uh flip. Yeah let me fix that quickly. No it's not default in composer, so we could make it default here.

Reply:
oh wait nvm, if it's not specified it wont end up in the yum config, and it will be enabled by default :facepalm:  Could you revert it to what it was? Sorry ._.

Reply:
Haha no problem